src/tree/param.h

```diff
@@ -2,7 +2,7 @@
  * Copyright 2014 by Contributors
  * \file param.h
  * \brief training parameters, statistics used to support tree construction.
- * \author Tianqi Chen
+n * \author Tianqi Chen
```

Comment:
Remove the stray letter in the beginning

Reply:
Done.